config-jetson.ini

```diff
@@ -83,3 +83,4 @@ TimeInterval = 0.5
 LogDirectory = /repo/data/processor/static/data
 EnableReports = no
 HeatmapResolution = 150,150
+WebHooksEndpoint=
```

Comment:
This has to be documented in one of these PRs, depending which gets merged first

Reply:
I included the `WebHooksEndpoint` documentantion in the README.